sql-plugin/src/main/scala/com/nvidia/spark/rapids/Arm.scala

```diff
@@ -15,6 +15,8 @@
  */
 package com.nvidia.spark.rapids
 
+import java.util
```

Comment:
I would remove this import. When reading Scala code below it's looks more useful to see explicitly that the method operates on Java API